misk/src/main/kotlin/misk/cloud/fake/security/keys/FakeKeyManager.kt

```diff
@@ -0,0 +1,33 @@
+package misk.cloud.fake.security.keys
+
+import misk.security.keys.KeyManager
+import okio.ByteString
+import java.nio.ByteBuffer
+import java.security.Key
+import javax.crypto.Cipher
+import javax.crypto.spec.SecretKeySpec
+
+internal class FakeKeyManager : KeyManager {
+    override fun encrypt(keyAlias: String, plainText: ByteString): ByteString {
+        val key = newSymmetricKey(keyAlias)
+        val cipher = Cipher.getInstance("AES")
+        cipher.init(Cipher.ENCRYPT_MODE, key)
+        return ByteString.of(ByteBuffer.wrap(cipher.doFinal(plainText.toByteArray())))
```

Comment:
I think we need an initialization vector? All I know about encryption is that it’s always harder than it should be. In this case cipher does the encryption but I recall that isn’t sufficient.

Reply:
Ahhh, this is just a fake. Never mind me.